include/wx/datetime.h

```diff
@@ -1565,6 +1567,18 @@ using wxDateTimeArray = wxBaseArray<wxDateTime>;
 //     virtual methods to work with the holidays they correspond to.
 // ----------------------------------------------------------------------------
 
+// Compares dates only on day and month.
+// Useful for finding dates within a map of holidays.
+class wxHolidayLess
```

Comment:
This doesn't seem used any longer now that we don't have a map any more, do you think it's still worth adding it? I'd rather omit it.

Reply:
Removed now.